ipld/plugin/nmt.go

```diff
@@ -406,3 +410,26 @@ func MustCidFromNamespacedSha256(hash []byte) cid.Cid {
 	}
 	return cidFromHash
 }
+
+// NamespacedSha256FromCID derives the Namespaced hash from the given CID.
+func NamespacedSha256FromCID(cid cid.Cid) []byte {
+	return cid.Hash()[mhOverhead:]
+}
+
+// RowMin returns the minimum NamespaceID in the given row.
+func RowMin(row []byte) []byte {
```

Comment:
These two functions basically return the left half or right half of a row? Why do they split the row in half, is this assuming that there is always only two IDs in a row? The way you are using it below in tests with `nID.Less()` would not be correct if the row had more than two namespaces I don't think?

Reply:
nvm, misunderstood what was being passed to these.

Reply:
Can I do it in a separate PR?

Reply:
since will have to wait on this to get merged. https://github.com/celestiaorg/nmt/pull/51

Reply:
i've merged it!

Reply:
Oh nice.